explanation/instances.md

```diff
@@ -56,19 +56,22 @@ When an instance is stopped, it executes the following steps in order:
 Beyond that, the instance will be removed from AMS either because you deleted it or because an error occurred during its runtime.
 ![Instance stop|575x521](https://assets.ubuntu.com/v1/abb5becf-instance_stop.png)
 
-### Possible instance status
-
-An instance moves through different stages and correspondingly can have the following status depending on its current state.
-
-Status            |  Description
-----------------|------------
-`created`     | AMS has created an internal database object for the instance and will next schedule the instance onto a suitable LXD node.
-`prepared` | AMS has decided the LXD node on which it will schedule the instance.
-`started` | The instance is started and now booting. During the boot sequence, possible hooks are executed. Only when all hooks have been executed, the instance will switch to `running`.
-`running` | The instance is fully up and running.
-`stopped` | The instance is fully stopped and will be deleted by AMS.
-`deleted` | The instance is deleted and will be removed from the AMS database soon.
-`error` | An error occurred while processing the instance. The instance is stopped. Further information about the error can be viewed with `amc show <instance id>`.
+### Possible instance statuses
+
+An instance moves through different stages and correspondingly can have the following statuses depending on its current state.
+
+| Status            |  Description |
+|-------------------|--------------|
+| `created`         | AMS has created an internal database object for the instance and will next schedule the instance onto a suitable LXD node. |
+| `prepared`        | AMS has decided the LXD node on which it will schedule the instance. |
+| `started`         | The instance is started and now booting. During the boot sequence, possible hooks are executed. Once all hooks have been executed, the instance will switch to `running`. |
+| `running`         | The instance is fully up and running. |
+| `stopped`         | The instance is fully stopped and will be deleted by AMS. |
```

Comment:
This is not necessarily true we can also do an `amc stop <cid>` to stop an instance and restart it again?